CHANGES.rst

```diff
@@ -209,6 +213,10 @@ Bug Fixes
 
 - ``astropy.io.fits``
 
+  - ``comments`` meta key is output to ``COMMENT`` instead of ``COMMENTS``
+    header. Similarly, ``COMMENT`` headers are read into ``comments`` meta
+    [#6097]
```

Comment:
> My inclination is to bump this to 2.0 and indicate as both a bug fix and API.

@taldcroft , so these duplicate entries in different sections are what you wanted?